src/MapUtil/MapUtil.js

```diff
@@ -393,6 +404,158 @@ export class MapUtil {
     return resolution >= layer.get('minResolution')
       && resolution < layer.get('maxResolution');
   }
+
+  /**
+   * Converts a given OpenLayers layer to a inkmap layer spec.
+   *
+   * @param {import("ol/layer/Layer").default} olLayer The layer.
+   *
+   * @return {Promise<import("../types").InkmapLayer | null>} Promise of the inmkap layer spec.
+   */
+  static async mapOlLayerToInkmap(olLayer) {
```

Comment:
This in turn I would expect to be made available through the 'LayerUtil`.

Reply:
moved to LayerUtil here: c25ebfe37849febc42098cf8604d458e51941901